extensions/core-cloud/runtime/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.camel.quarkus</groupId>
+        <artifactId>camel-quarkus-core-cloud-parent</artifactId>
+        <version>0.1.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>camel-quarkus-core-cloud</artifactId>
```

Comment:
Why the name `camel-quarkus-core-cloud`? Are we going to have a separate `camel-quarkus-cloud` one day? 

The current one depends on `camel-cloud` which I assume is a Camel component, so why is this not called simply `camel-quarkus-cloud`?

Reply:
No, this is not a camel component but the implementation of the `org.apache.camel.cloud` interfaces that are part of camel-core-api